integration_tests/test_rpm_source.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2016 Neal Gompa
+# Copyright (C) 2016, 2017 Neal Gompa
```

Comment:
well since I started a review, please make this
`# Copyright (C) 2016-2017 ...`